src/frontend/main.js

```diff
@@ -87,12 +90,23 @@ $(function() {
     };
 
     const removeBookmark = function() {
-        const bookmark = results.find('.selected');
-        const id = bookmark.data('id');
-        bookmarksCollection.remove(id, function() {
-            selectNext();
-            bookmark.remove();
-            loadBookmarks();
+        sweetAlert({
```

Comment:
may be move configured sweetAlert into function? confirmBookmarkDelete or smth like